client/extensions/woocommerce/app/promotions/promotions-list-row.js

```diff
@@ -30,22 +34,22 @@ function getTimeframeText( promotion, translate, moment ) {
 	if ( promotion.startDate && promotion.endDate ) {
 		return translate( '%(startDate)s - %(endDate)s', {
 			args: {
-				startDate: moment( promotion.startDate + 'Z' ).format( 'll' ),
-				endDate: moment( promotion.endDate + 'Z' ).format( 'll' ),
+				startDate: moment( promotion.startDate ).format( 'll' ),
```

Comment:
No more Z / gmt needed?

Reply:
The API doesn't let me send in `_gmt` fields, I have to deal with everything in the local timezone of the store. ☹️